### PR TITLE
arangodb 3.10.0

### PR DESCRIPTION
**Pull request by cho-m**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by cho-m** [comment]
I think this needs some changes for ARM build, so adding `do not merge`.

File: Formula/arangodb.rb [hunk 1]
```diff
@@ -51,18 +51,27 @@ def install
       system "go", "build", *std_go_args(ldflags: ldflags), "github.com/arangodb-helper/arangodb"
     end
 
+    arch = case Hardware.oldest_cpu
+    when :arm_vortex_tempest
+      "neon"
+    when :core2
+      "merom" # only supports `core`, `merom`, and `penryn`
```
Comment (cho-m) [inline Formula/arangodb.rb:58]: Previous attempt at building older Core2 failed (#99988) so may not be well supported. May end up having to compile with at least `nehalem` again.

Comment (cho-m) [reply Formula/arangodb.rb]: Works now. Using `merom` to enable SSSE3 as `core` only allows up to SSE3.

File: Formula/arangodb.rb [hunk 2]
```diff
@@ -51,18 +67,27 @@ def install
...
-      -DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}
-      -DUSE_CATCH_TESTS=Off
-      -DUSE_GOOGLE_TESTS=Off
       -DCMAKE_INSTALL_LOCALSTATEDIR=#{var}
+      -DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}
+      -DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}
+      -DTARGET_ARCHITECTURE=#{arch}
+      -DUSE_GOOGLE_TESTS=OFF
+      -DUSE_JEMALLOC=OFF
+      -DUSE_MAINTAINER_MODE=OFF
```
Comment (cho-m) [inline Formula/arangodb.rb:89]: Mostly reordered. I did drop `-DUSE_CATCH_TESTS=Off` as it wasn't being used.

**Comment by cho-m** [comment]
ARM built but test is failing
```
==> /opt/homebrew/Cellar/arangodb/3.10.0/bin/arangosh --server.password "" --javascript.execute-string "require('@arangodb').print('it works!')"
  cannot find configuration file

(tried locations: '/private/tmp/arangodb-test-20221005-66952-7z6gok/etc/relative/arangosh.conf', '/opt/homebrew/Cellar/arangodb/3.10.0/etc/opt/homebrew/Cellar/arangodb/3.10.0/arangodb3/arangosh.conf', '/private/tmp/arangodb-test-20221005-66952-7z6gok/arangosh.conf', '/private/tmp/arangodb-test-20221005-66952-7z6gok/.arangodb/arangosh.conf', '/etc/opt/homebrew/Cellar/arangodb/3.10.0/arangodb3/arangosh.conf')
```

---

Other than HOME-based paths, others are a bit odd:
- /opt/homebrew/Cellar/arangodb/3.10.0/etc/opt/homebrew/Cellar/arangodb/3.10.0/arangodb3/arangosh.conf
- /etc/opt/homebrew/Cellar/arangodb/3.10.0/arangodb3/arangosh.conf

May need to check on how `etc` dir is identified

**Review by carlocab — APPROVED** [review]

**Comment by carlocab** [comment]
Intel Big Sur is about 15% faster than Catalina at `brew install --verbose --build-bottle arangodb`. Probably mostly due to PGO/LTO. Nice.

**Pull request closed without merging**